Sources/AblyAssetTrackingInternal/Configuration/Host.swift

```diff
@@ -0,0 +1,12 @@
+/// Internal-only type for configuring some properties of `ARTClientOptions` which we do not wish to expose through the public `ConnectionConfiguration` interface. Intended for use in testing.
+public struct Host {
```

Comment:
If this is internal only, should its accessibility modifier be internal (or omitted as that's the default)?

Reply:
If it were `internal`, that would mean that it was only accessible inside the `AssetTrackingInternal` library – but we need it to be visible to callers of the `public` method `create(configuration:,host:,logHandler:)`.

It's a bit confusing — the `AssetTrackingInternal` library is intended only to be used internally by the SDK, but since the rest of the SDK lives in other libraries, all of the functionality it wishes to expose to the rest of the SDK needs to be public.

Reply:
Ah yep, gotcha!